src/summary/history/notes-card-style.css

```diff
@@ -1,18 +1,32 @@
-.table {
-  width: 100%;
+.moreNotes {
+  display: flex;
+  align-items: center;
+  border: none;
+  padding: 0.625rem 1.625rem 0.625rem 1rem;
+  background-color: var(--omrs-color-bg-medium-contrast);
+  color: var(--omrs-color-ink-medium-contrast);
+  cursor: pointer;
+  text-decoration: none;
 }
 
-.tableRow {
+.tableNotesRow {
```

Comment:
I dont see why you had to rename the classes here

Reply:
It tends to be affected by other classes such as dimensions

Reply:
so making it achieve the Figma design responsiveness on phones affects other files classes. i.e display: block;